### PR TITLE
Showcase: convert ShwText components to `.gts`

### PR DESCRIPTION
**Pull request by shleewhite**
### :pushpin: Summary

If merged, this PR would convert the ShwText components to the new `.gts` format.

Additional changes:
* updates the `ember-truth-helpers` package because it is necessary to import `eq` into `.gts` files
* disable the `'ember/no-empty-glimmer-component-classes'` rule because ember doesn't support template only components that have type signatures

### :link: External links

Jira ticket: [HDS-3838](https://hashicorp.atlassian.net/browse/HDS-3838)

***

### 👀 Component checklist

- [x] Percy was checked for any visual regression

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

[HDS-3838]: https://hashicorp.atlassian.net/browse/HDS-3838?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by aklkv — APPROVED** [review]
Great work 🎉 I might be helpful to spit out `ember-truth-helpers` upgrade

File: showcase/.eslintrc.js [hunk 1]
```diff
@@ -26,6 +26,8 @@ module.exports = {
         functions: 'only-multiline',
       },
     ],
+    // ember doesn't support empty component classes that have type signatures yet
```
Comment (didoo) [inline showcase/.eslintrc.js:29]: [praise] 👏 (for the clear comment)

File: showcase/app/components/shw/text/index.gts [hunk 2]
```diff
@@ -4,8 +4,9 @@
  */
 
 import Component from '@glimmer/component';
-import { action } from '@ember/object';
 import { assert } from '@ember/debug';
+import { eq } from 'ember-truth-helpers';
```
Comment (didoo) [inline showcase/app/components/shw/text/index.gts:8]: [thought] nice! didn't realize modifiers should be imported this way (makes sense, now that I see it)

**Review by didoo — APPROVED** [review]
spotless, as usual 👍

**Pull request merged**